Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppProtectionPolicyAndroid/MSFT_IntuneAppProtectionPolicyAndroid.psm1

```diff
@@ -884,6 +998,9 @@ function Test-TargetResource
     # wipe out the current apps value if AppGroupType is anything but selectedpublicapps to match the appshash values
     if ($CurrentValues.AppGroupType -ne 'selectedPublicApps') { $CurrentValues.Apps = @() }
 
+    # remove thre ID from the values to check as it may not match
+    $targetvalues.remove('ID')
```

Comment:
Please pipe with | Out-Null when removing values. Otherwise these will attempt to throw a verbose message and sometimes cause conflict with the export feature.